smart-contracts/assembly/contracts/FT/WMAS.ts

```diff
@@ -33,9 +46,20 @@ export function withdraw(bs: StaticArray<u8>): void {
   const recipient = new Address(
     args.nextString().expect('recipient is missing'),
   );
-
-  assert(amount > 0, 'Payment must be more than 0 WMAS');
-
   burn(u256ToBytes(u256.fromU64(amount)));
   transferCoins(recipient, amount);
 }
+
+function computeStorageCost(receiver: Address): u64 {
+  let cost = 0;
+  if (!Storage.hasOf(Context.callee(), balanceKey(receiver))) {
```

Comment:
Inversing the if condition will make code more elegant:
```suggestion
if (Storage.hasOf(Context.callee(), balanceKey(receiver))) {
  return 0;
}
```